python-package/geobr/lookup_muni.py

```diff
@@ -1,35 +1,44 @@
-""" Lookup municipality codes and names
 
-Input a municipality NAME or CODE and get the names and codes of the municipality's corresponding state, meso, micro,
-intermediate, and immediate regions.
+import pandas as pd
+import utils
 
-:param name_muni -- The municipality name to be looked up
-:param code_muni -- The municipality code to be looked up
 
-:return data.frame with 13 columns identifying the geographies information of that municipality
+def lookup_muni(name_muni='', code_muni='', verbose=False):
```

Comment:
I have a suggestion for the conditions flow:

First, set the default to `None`
`name_muni=None` and `code_muni=None`

```python
if name_muni is None and code_muni is None:
     return <all data>
elif name_muni is not None and code_muni is not None:
     raise <Please insert either a municipality name or a municipality code>
elif name_muni is not None:
    return <filter by name muni>
elif cod_muni is not None:
   return <filter by cod_muni>
```

I think it is more pythonic than defaulting a variable to `''` and the default is to get all table.

Also, the docstring should warn the user that you should not select both `cod_muni` and `name_muni`